content/page/get-involved.md

```diff
@@ -0,0 +1,64 @@
+---
+author: kushinauwu
+cover:
+  alt: obf_membership
+  image: /wp-content/uploads/2019/02/obf_membership.jpg
+date: "2019-02-04T19:04:53+00:00"
+guid: https://www.open-bio.org/wp/?page_id=2546
+title: Get Involved
```

Comment:
```suggestion
title: Get Involved with OBF
```

Reply:
i like pages to be fully identified so i don't have a bunch of tabs that all say "Get Involved" (from different sites), but i don't feel strongly about this.

Reply:
So no change needed?